packages/@aws-cdk/toolkit-lib/lib/actions/bootstrap/index.ts

```diff
@@ -169,6 +169,18 @@ export interface BootstrapParameters {
   readonly customPermissionsBoundary?: string;
 }
 
+export interface EnvironmentBootstrapResult {
+  environment: cxapi.Environment;
+  status: 'success' | 'no-op';
+  message?: string;
```

Comment:
if we keep message, it should not be optional since we always set it anyway.

Reply:
Removed it as per the change in toolkit.ts.